pyomo/contrib/incidence_analysis/interface.py

```diff
@@ -180,12 +187,17 @@ def __init__(self, model=None, active=True, include_fixed=False):
                     (var, idx) for idx, var in enumerate(self.variables))
             self.con_index_map = ComponentMap(
                     (con, idx) for idx, con in enumerate(self.constraints))
-            self.incidence_matrix = nlp.evaluate_jacobian_eq()
+            if include_inequality:
+                self.incidence_matrix = nlp.evaluate_jacobian()
+            else:
+                self.incidence_matrix = nlp.evaluate_jacobian_eq()
```

Comment:
This changes behavior. Previously the nlp interface would always ignore inequalities, and the model interface would never ignore inequalities. This patch makes the two interfaces consistent (by default, include inequalities) and adds the option to ignore inequalities, which applies to both interface.